.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+
+# command to install dependencies
+install:
+  - "pip install ."
```

Comment:
I don't quite follow - this was taken from `courseraprogramming`: https://github.com/coursera/courseraprogramming/blob/master/.travis.yml#L7

What's the difference between using `pip` and `setup.py` for travis-ci?